pytest_localftpserver/plugin.py

```diff
@@ -13,6 +15,11 @@
 
 import pytest
 
+if sys.platform.startswith('linux'):
```

Comment:
As I said, for some reason the tests on OSX didn't stop the Process properly, so `test_ftp_stopped` would run "forever". Thats why I used Process for linux only.

Reply:
How about the compromise, that you merge the PR and open an issue for OSX which someone who is more experienced with `Process` subclasses can  fix. Since i don't have a Mac i can only trust the Travis tests, which timeout if `Process` is used.

Reply:
I think you are correct. MacOS does something weird, and I can't verify it too.